### ISSUE TITLE
starting kubernetes always on status "kubernetes starting"

### ISSUE THREAD

**Issue by PeterLappo** [docker/for-mac#2568]
### Expected behavior
Docker with Kubernetes
### Actual behavior
Docker starts but Kubernetes is starting
### Information

  - Full output of the diagnostics from "Diagnose & Feedback" in the menu
Docker for Mac: version: 18.02.0-ce-rc2-mac51 (1179f458c5db5072781101144a8d7e017fcb568c)
macOS: version 10.12.6 (build: 16G1212)
logs: /tmp/14DBF6F7-32DE-4921-8318-92D2AC2B38BB/20180205-214849.tar.gz
[20180205-214849.tar.gz](https://github.com/docker/for-mac/files/1696942/20180205-214849.tar.gz)

[OK]     db.git
[OK]     vmnetd
[OK]     dns
[OK]     driver.amd64-linux
[OK]     virtualization VT-X
[OK]     app
[OK]     moby
[OK]     system
[OK]     moby-syslog
[OK]     kubernetes
[OK]     env
[OK]     virtualization kern.hv_support
[OK]     slirp
[OK]     osxfs
[OK]     moby-console
[OK]     logs
[OK]     docker-cli
[OK]     menubar
[OK]     disk

  - A reproducible case if this is a bug, Dockerfiles FTW
  - Page URL if this is a docs issue or the name of a man page

diagnostic id 14DBF6F7-32DE-4921-8318-92D2AC2B38BB/20180205-214849

### Steps to reproduce the behavior

  1. Uninstall Docker for mac
  2. Install Docker for mac
  3. enable Kubernetes

Duplicate of https://github.com/docker/for-mac/issues/2559

**Comment by PeterLappo** [comment]
Reset to factory defaults and re-enabling Kubernetes solved this problem

**Comment by pgayvallet** [comment]
Was it a fresh install (just before failure) or were you updating from previous rc/edge ?

**Comment by PeterLappo** [comment]
Slightly more complicated. I had installed docker kubernetes about two weeks ago (not sure which version). My mac upgraded itself and rebooted while docker / kubernetes was running. After the reboot a pod i was running could not be deleted (using kubectl) and was draining my battery so I installed the version above, at which point kubernetes would not start. Eventually I did a factory reset which resulted in kubernetes starting.
